src/index.js

```diff
@@ -113,30 +133,23 @@ async function deleteDB() {
         });
 }
 
-function getExpireKey(store, key) {
-    return store + EXPIRE_GLUE + key;
-}
-
 async function get(store, key, validate = true) {
     const { db } = cacheInstance;
-
-    const expired = await db[EXPIRE_BIN].get(getExpireKey(store, key));
-    const isValid = validate && db[store] ? expired > Date.now() : true;
+    const { value = null, expire = 0 } = await db[store].get(key) || {};
+    const isValid = validate ? expire > Date.now() : true;
 
     if (!isValid) {
-        await db[EXPIRE_BIN].delete(getExpireKey(store, key));
+        await db[store].delete(key);
     }
 
-    return isValid ? await db[store].get(key) : null;
+    return isValid ? value : null;
 }
 
-async function set(store, key, value, expire = Number.MAX_SAFE_INTEGER) {
+async function set(store, key, value, ttl = Number.MAX_SAFE_INTEGER) {
     const { db } = cacheInstance;
+    const expire = Date.now() + Number(ttl);
```

Comment:
Number('fdfff') // NaN

I think it would be better to check its type before

Reply:
I think it would be better to rewrite it to typescript. The check doesn't help at all.